### PR TITLE
Add integer types

### PR DESCRIPTION
**Pull request by lukewagner**
As discussed in #64.

### CODE REVIEW

File: proposals/interface-types/Explainer.md [hunk 1]
```diff
@@ -488,10 +489,54 @@ representation choices to be added over time. Moreover, the provider module is
...
+> **NOTE** In the future we could consider supporting the more general set of
+> (u|s)\<bitwidth\> types, supporting more precise static interface contracts.
+> The current set is chosen for it's practical application to C and Web IDL.
+
+
 ### TODO
 
 This rough list of topics is still to be added in subsequent PRs:
-* integers and bool
+* bool
```
Comment (jgravelle-google) [inline proposals/interface-types/Explainer.md]: `u1` and done?

Comment (fgmccabe) [reply proposals/interface-types/Explainer.md]: Umm. Why bool not an enumeration type?

Comment (jgravelle-google) [reply proposals/interface-types/Explainer.md]: Mostly because I forgot that was an option.

Nevermind!

**Comment by Hywan** [comment]
Looks good to me :-).

**Pull request merged**